Tests/GeoLib/TestLineSegmentIntersect2d.cpp

```diff
@@ -50,18 +50,29 @@ class LineSegmentIntersect2dTest : public testing::Test
 	ac::gtest_reporter gtest_reporter;
 };
 
+#if not defined(_MSC_VER) && _MSC_VER < 1900
+// Compilers of MVS below 2015 do not support unrestricted unions. The
+// unrestricted union is used by autocheck to handle test data. The autocheck
+// workaround for MVS compilers (below version 2015) contains a bug and in the
+// consequence the tests crashes. For this reason the tests are disabled under
+// this environments.
```

Comment:
Autocheck needs the default constructor when it doesn't handle the unrestricted union properly.

> I think it's a bit unfortunate that our unit tests have higher demands on the compiler than the code they test...

Thats true. But I think this was not intended when choosing autocheck.